recipes/xmscore/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "xmscore" %}
+{% set version = "3.0.7" %}
+{% set sha256 = "98acd332ac5a14602539e218db82b2970dbeed2b149e0d47dcbd80c6778801c7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Aquaveo/{{ name|lower }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.0.0
+    - boost =1.67
```

Comment:
conda-forge has a global pinning for `boost 1.70.0`. If this really is limited to `1.67` people won't be able to install this package along side the rest of the conda-forge stack that uses boost. If this is not a mandatory limitation I recommend you to just remove the pin and let our infrastructure take care of that.